ultralytics/engine/exporter.py

```diff
@@ -359,12 +359,12 @@ def get_int8_calibration_dataloader(self, prefix=""):
             task=self.model.task,
             imgsz=self.imgsz[0],
             augment=False,
-            batch_size=self.args.batch * 2,  # NOTE TensorRT INT8 calibration should use 2x batch size
+            batch_size=self.args.batch,
         )
         n = len(dataset)
         if n < 300:
             LOGGER.warning(f"{prefix} WARNING ⚠️ >300 images recommended for INT8 calibration, found {n} images.")
-        return build_dataloader(dataset, batch=self.args.batch * 2, workers=0)  # required for batch loading
```

Comment:
Same as above, I think we need this 2x batch size for TRT, and for others should not impact INT8 quantization, or may partly speed it up.

Reply:
Same reason as above!